### ISSUE TITLE
Fix failing v2-beta term frequencies test

### ISSUE THREAD

**Issue by johnbradley** [phenoscape/rphenoscape#213]
The `obtaining/calculating term frequencies` test is failing when using the [v2-beta kb.phenoscape API](https://dev.phenoscape.org/api/v2-beta)
```
Failure (test-freqs.R:119:3): obtaining/calculating term frequencies
all(wt < wt1) is not TRUE
```
https://github.com/phenoscape/rphenoscape/blob/b668eb99c7fd29dc24a2ee6dd4c7f19a48926aca/tests/testthat/test-freqs.R#L110-L119
The error is because the v2-beta API returns an additional `pelvic fin pointed` phenotype from `get_phenotypes()`.
This phenotype has a frequency of 0 for corpus="taxa" and frequency of 7.308731e-06 for corpus="taxon_annotations".  
```
> iri <- "http://purl.org/phenoscape/expression?value=%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000051%3E+some+%0A++++%28%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0002258%3E%0A+++++and+%28%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FRO_0000052%3E+some+%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0000152%3E%29%29"
> term_freqs(iri, as = "phenotype", corpus = "taxon_annotations")
[1] 7.308731e-06
> term_freqs(iri, as = "phenotype", corpus = "taxa")
[1] 0
```

**Comment by hlapp** [comment]
@balhoff can you comment under which legitimate circumstances there could be phenotype with #annotations > 0 yet it is not linked to any taxon profile (which I believe is what we're looking at for corpus 'taxa', right?). My sense is that it's possible, but without delving into the algorithm I'm not sure what those cases would need to be like.

**Comment by balhoff** [comment]
It took a little bit of digging, but I think it does make sense. This phenotype is associated with a character state that has no alternative state (it's the only state for its character). The algorithm works with character state variation first and then accesses the associated phenotypes.

**Comment by hlapp** [comment]
So when the algorithm sees only one state for a character for some but not all of a node's children, the assumption is that all siblings will have the same state rather than that at least some of the siblings have a different state except we're missing the data for them? I somehow would have thought it's the other way around, but I guess not. I suppose the idea is to be conservative and only infer variation where data is there to support it?

**Comment by hlapp** [comment]
Anyway, I think @johnbradley the solution would have to be to relax the test so as to not require _every_ phenotype to have a non-zero frequency for the 'taxa' corpus?

**Comment by johnbradley** [comment]
Fixed by PR #221
